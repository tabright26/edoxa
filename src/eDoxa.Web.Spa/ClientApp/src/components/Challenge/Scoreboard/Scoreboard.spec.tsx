// import React from "react";
// import renderer from "react-test-renderer";
// import { Provider } from "react-redux";
// import { MemoryRouter } from "react-router-dom";
// import Scoreboard from "./Scoreboard";
// import { ChallengesState } from "store/root/challenge/types";

it("renders without crashing", () => {
  // //Arrange
  // const challenges: ChallengesState = {
  //   data: [
  //     {
  //       id: "123",
  //       name: "",
  //       state: "Inscription",
  //       bestOf: 3,
  //       entries: 10,
  //       game: "LeagueOfLegends",
  //       payoutEntries: 5,
  //       entryFee: { currency: "token", amount: 0 },
  //       participants: [
  //         { id: "id1", challengeId: "123", score: 0, matches: [] }
  //       ],
  //       timeline: {
  //         createdAt: 123123123,
  //         startedAt: null,
  //         endedAt: null,
  //         closedAt: null
  //       },
  //       scoring: new Map<string, string>(),
  //       payout: {
  //         prizePool: { currency: "token", amount: 200000 },
  //         buckets: []
  //       }
  //     },
  //     {
  //       id: "456",
  //       name: "",
  //       state: "Inscription",
  //       bestOf: 3,
  //       entries: 10,
  //       game: "LeagueOfLegends",
  //       payoutEntries: 5,
  //       entryFee: { currency: "token", amount: 0 },
  //       participants: [
  //         { id: "id2", challengeId: "456", score: 0, matches: [] }
  //       ],
  //       timeline: {
  //         createdAt: 123123123,
  //         startedAt: null,
  //         endedAt: null,
  //         closedAt: null
  //       },
  //       scoring: new Map<string, string>(),
  //       payout: {
  //         prizePool: { currency: "token", amount: 200000 },
  //         buckets: []
  //       }
  //     },
  //     {
  //       id: "678",
  //       name: "",
  //       state: "Inscription",
  //       bestOf: 3,
  //       entries: 10,
  //       game: "LeagueOfLegends",
  //       payoutEntries: 5,
  //       entryFee: { currency: "token", amount: 0 },
  //       participants: [
  //         { id: "id3", challengeId: "678", score: 0, matches: [] }
  //       ],
  //       timeline: {
  //         createdAt: 123123123,
  //         startedAt: null,
  //         endedAt: null,
  //         closedAt: null
  //       },
  //       scoring: new Map<string, string>(),
  //       payout: {
  //         prizePool: { currency: "token", amount: 200000 },
  //         buckets: []
  //       }
  //     }
  //   ],
  //   loading: false,
  //   error: null
  // };

  // const store: any = {
  //   getState: () => {
  //     return {
  //       root: {
  //         challenge: challenges
  //       }
  //     };
  //   },
  //   dispatch: action => {},
  //   subscribe: () => {}
  // };

  // //Act
  // const tree = renderer
  //   .create(
  //     <Provider store={store}>
  //       <MemoryRouter>
  //         <Scoreboard />
  //       </MemoryRouter>
  //     </Provider>
  //   )
  //   .toJSON();

  // //Assert
  // expect(tree).toMatchSnapshot();
});
