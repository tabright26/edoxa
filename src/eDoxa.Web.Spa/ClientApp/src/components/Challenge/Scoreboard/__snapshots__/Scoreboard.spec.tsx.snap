// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <div
    bg="dark"
    className="my-2 text-light text-center card"
  >
    <div
      as="h5"
      className="border-0 card-header"
    >
      Scoreboard
    </div>
  </div>,
  <div
    aria-expanded="false"
    className="collapse"
    style={Object {}}
  >
    <div
      className="text-center mt-5 card-body"
    >
      <div
        className="animated fadeIn pt-1 text-center"
      >
        <div
          className="sk-spinner sk-spinner-pulse"
        />
      </div>
    </div>
  </div>,
]
`;
